student.rb

```diff
@@ -3,13 +3,17 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+  def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
![image](https://github.com/Eccb7/school-library/assets/93039359/2c11c837-a878-44a3-a5ec-2a23a86418a8)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you put back the equal sign `=` back after the name and also put it after the parent_permission  because I am getting this error while creating a student if I can't create a student then I can't put a student in a classroom or a classroom to a student so kindly make your initialize as below everywhere ⬇️ It was already broken but it would be nice if put equal sign at before name and parent_permission in all the classes person, student and teacher 👍 to make your app work without any errors 😊

`def initialize(age, classroom, name = 'Unknown', parent_permission= true)`